mocks/register-form.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ingresar</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="index.css">
+    <style>
+        body {
+            background: linear-gradient(-45deg, #424116, #194111, #0a2833, #630f0f);
+            background-size: 400% 400%;
+            height: 100vh;
+        }
+    </style>
+</head>
+
+<body class="animated">
+    <div class="container">
+        <div class="logotype">👋 rar.vg</div>
+        <div class="login-form">
+            <h1 class="l p-no-margin-top">Sign up</h1>
+            <input type="text" placeholder="Name">
+            <input type="text" placeholder="Username">
```

Comment:
I'd add something that suggests that the chosen username corresponds to the assigned subdomain.
Maybe incorporate this? https://stackoverflow.com/a/43964636

Reply:
Also, doing this would be a cool QoL change.
```suggestion
            <input type="text" placeholder="Username / Domain">
```